tests/test_fips.py

```diff
@@ -0,0 +1,113 @@
+# Tests for FIPS-related commands.
+
+from testing import UbuntuAdvantageTest
+
+
+class FIPSTest(UbuntuAdvantageTest):
+
+    SERIES = 'xenial'
+
+    def test_enable_fips(self):
+        """The enable-fips option enables the FIPS repository."""
+        p = self.boot_cfg
+        p.write_text("")
```

Comment:
`p` is unneded, please just `self.boot_cfg.write_text("")` (same for other tests below)